clouditor-engine-core/src/main/java/io/clouditor/assurance/Certification.java

```diff
@@ -133,4 +122,52 @@ public String getWebsite() {
   public void setWebsite(String website) {
     this.website = website;
   }
+
+  @Override
+  public String toString() {
+    return "Certification{"
+        + "id='"
+        + id
+        + '\''
+        + ", controls="
+        + controls
+        + ", description='"
+        + description
+        + '\''
+        + ", publisher='"
+        + publisher
+        + '\''
+        + ", website='"
+        + website
+        + '\''
+        + '}';
+  }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) return true;
+
+    if (o == null || getClass() != o.getClass()) return false;
+
+    Certification that = (Certification) o;
+
+    return new EqualsBuilder()
+        .append(getId(), that.getId())
+        .append(new ArrayList<>(getControls()), new ArrayList<>(that.getControls()))
```

Comment:
?!??!?!

Reply:
It is to genearate equality between a org.hibernate.collection.internal.PersistentBag and a java.util.List